### PR TITLE
docs(producer): update content

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### Update readme

Restructure the order of the workshop

### CODE REVIEW

File: producer-node-lab.md [hunk 1]
```diff
@@ -1,60 +1,53 @@
...
-
-## Node Configuration Steps
-
+Start your own single-node blockchain with single steps:
 
 - Create a genesis json file 
 - Configure the producer node 
-- Logging JSON 
-- Log Rotate
+- Logging JSON
```
Comment (leisterfrancisco) [inline producer-node-lab.md:31]: @xavier506 not sure about it

**Review by xavier506 — APPROVED** [review]
Awesome Work leister

File: network-boot-lab.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+At the time of configuring a private EOSIO network there are some software installation requirements:
+1. EOSIO precompiled binaries installation
+1. EOSIO.CDT binaries installation
+1. cmake
+1. build-essential
+
+*Also, we are using Ubuntu 18.04 LTS, but if you want to use macOS or other supported OS, take a look at [EOSIO releases](https://github.com/EOSIO/eos/releases)*
+
+### EOSIO Precompiled Binaries Installation
+Execute the following commands to install EOSIO precompiled binaries:
```
Comment (xavier506) [inline network-boot-lab.md:30]: this works on debian linux platforms , what if the user is on a mac or windows or centos linux?

File: network-boot-lab.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+    ```
+## `eosio.bios` Contract Deployment
+### Activate `PREACTIVATE_FEATURE`
+Due to updates introduced in v1.8 and v2.0, it is necessary to activate a protocol feature called PREACTIVATE_FEATURE. To activate this feature, execute the following command:
+```bash
+$ curl --request POST \
+	--url http://localhost:8888/v1/producer/schedule_protocol_feature_activations \
+	-d '{"protocol_features_to_activate": ["0ec7e080177b2c02b278d5088611686b49d739925a92d9bfcacd7fc6b74053bd"]}'
+```
+### Deploy **Old** Contracts Version
```
Comment (xavier506) [inline network-boot-lab.md:211]: is this still required in the latest version of eosio?

File: producer-node-lab.md [hunk 4]
```diff
@@ -1,60 +1,58 @@
...
+1. The config.ini file used to run nodeos.
+2. The command used to run nodeos from terminal with the needed plugins (`.txt` file extention).
 
-## Before you begin
+### Help material
 
-* [Install the EOSIO software](https://developers.eos.io/manuals/eos/latest/install/index) before starting this section.
-* It is assumed that `nodeos`, `cleos`, and `keosd` are accessible through the path. If you built EOSIO using shell scripts, make sure to run the [Install Script](https://developers.eos.io/manuals/eos/latest/install/build-from-source/shell-scripts/install-eosio-binaries).
-* Know how to pass [Nodeos options](https://developers.eos.io/manuals/eos/latest/nodeos/usage/nodeos-options) to enable or disable functionality.
+To successful complete this workshop the student can use next the help links:
```
Comment (xavier506) [inline producer-node-lab.md:13]: To successfully complete...

File: producer-node-lab.md [hunk 5]
```diff
@@ -1,60 +1,58 @@
 # Run a Local Producer Node
-## Goal
 
-This section describes how to set up a single-node blockchain configuration running on a single host.  This is referred to as a _**single host, single-node testnet**_.  We will set up one node on your local computer and have it produce blocks.  The following diagram depicts the desired single host testnet.
+## Workshop instructions
 
-![Single host single node testnet](https://developers.eos.io/315123127612b3c9153341b9e7401d02/single-host-single-node-testnet.png)
+Complete the step two by finding the missing plugin to configure nodeos using the config.ini file and also the console inline configuration.
```
Comment (xavier506) [inline producer-node-lab.md:5]: Complete step two ...

File: producer-node-lab.md [hunk 6]
```diff
@@ -1,60 +1,58 @@
...
+Start your own single-node blockchain:
 
-- Create a genesis json file 
+- Create a `config.ini` file 
 - Configure the producer node 
-- Logging JSON 
-- Log Rotate
 
-The more advanced user will likely have need to modify the configuration.  `nodeos` uses a custom configuration folder.  The location of this folder is determined by your system.
+The more advanced user will likely have need to modify the configuration. `nodeos` uses a custom configuration folder that the location is determined by your system.
```
Comment (xavier506) [inline producer-node-lab.md:36]: you can override this is you set a config dir

File: producer-node-lab.md (hunk 1 above)
Comment (xavier506) [reply producer-node-lab.md]: lets remove it for now

**Comment by xavier506** [comment]
@leisterfrancisco  I will merge this PR now but please have a look at my comments and include the correction is a future PR

File: network-boot-lab.md (hunk 2 above)
Comment (leisterfrancisco) [reply network-boot-lab.md:30]: Solved

File: network-boot-lab.md (hunk 3 above)
Comment (leisterfrancisco) [reply network-boot-lab.md:211]: Solved

**Pull request merged**